R/tm_a_regression.R

```diff
@@ -91,6 +91,9 @@ tm_a_regression <- function(label = "Regression Analysis",
                             default_plot_type = 1,
                             default_outlier_label = "USUBJID") {
   logger::log_info("Initializing tm_a_regression")
+  if (!requireNamespace("methods", quietly = TRUE)) {
```

Comment:
so methods is always available so we don't need this

Reply:
updated.